packages/vue_frontend_rpc/src/components/Editor.vue

```diff
@@ -103,19 +82,18 @@ export default {
     },
     prepareTask(task) {
       this.task = task;
-      for (const question of task.content) {
+      const questions = [...task.content];
+      for (const question of questions) {
         for (const prop in question) {
           if (question[prop] === FUNCTION) {
-            const that = this;
-            question[prop] = this.getEvaluationFunction(that.rpc, question.name, prop);
+            var that = this;
+            question[prop] = (...args) => {
+              return that.rpc.invoke("evaluateMethod", [question.name, prop, args]);
```

Comment:
Is the order of the arguments here as expected?

Reply:
yes, you are right i made a change